proposals/lip-0043.md

```diff
@@ -240,15 +246,15 @@ def execute(trs: Transaction) -> None:
             "height": 0,
             "timestamp": 0,
             "stateRoot": EMPTY_HASH,
-            "validatorsHash": computeValidatorsHash(trs.params.initValidators, trs.params.certificateThreshold)
+            "validatorsHash": computeValidatorsHash(trs.params.sidechainValidators, trs.params.sidechainCertificateThreshold)
```

Comment:
`computeValidatorsHash()` not defined (previously it was bft. but now we should link to LIP0058 where it is defined). Same in the execution of the other command (line 523)

Reply:
It is defined at the beginning of the specifications section.